src/content/docs/configuration/boot_manager_configuration.md

```diff
@@ -90,8 +90,138 @@ Every time we modify the GRUB configuration file, we need to remake the config w
+default_entry: 2 # Points directly to the 'linux-cachyos' entry below
+
+/+CachyOS        # Entry 1: A directory (use /+ to expand by default)
+//linux-cachyos  # Entry 2: The actual bootable entry
+    protocol: linux
+    kernel_path: boot():/vmlinuz-linux-cachyos
+    cmdline: quiet splash root=UUID=... rw # Basic kernel parameters
+    module_path: boot():/initramfs-linux-cachyos.img
+```
+
+:::note
+`boot():/` refers to the root of the boot drive.
+:::
+
+### Theming
+
+You can customize the visual appearance of the Limine boot menu:
+
+* **Wallpaper:** Set a background image. Supported formats include BMP, PNG, and JPEG.
+  ```shell
+  # /boot/limine.conf
+
+  wallpaper: boot():/splash.png
+  wallpaper_style: stretched # Options: 'stretched', 'tiled', 'centered'
+  backdrop: 000000           # Background color (RRGGBB hex) if style is 'centered'
+  ```
+* **Fonts:** Use a [custom font file](https://github.com/viler-int10h/vga-text-mode-fonts) and adjust its size.
+  ```shell
+  # /boot/limine.conf
+
+  term_font: boot():/custom_font.F16
+  term_font_scale: 2x2 # Scales font size, useful for high-resolution displays
+  ```
+* **Colors:** Modify terminal text and background colors.
+  ```shell
+  # /boot/limine.conf
+
+  term_background: 80000000 # Example: Semi-transparent black (AARRGGBB)
+  # Other color options like term_foreground, etc., are available.
+  ```
+
+### Kernel Command Configuration
+
+On CachyOS with the Limine bootloader, you can configure the kernel parameters (also known as the kernel command line) passed during boot. The method differs slightly depending on whether your system uses UEFI or legacy BIOS firmware.
+
+#### UEFI Systems
+
+On UEFI systems, CachyOS utilizes the `limine-entry-tool` utility to **automatically manage kernel entries and parameters** based on a configuration file.
+
+1.  **Edit the configuration file:** Modify the `KERNEL_CMDLINE` variables in `/etc/default/limine`. You can set default parameters for all kernels or specific parameters for certain kernel names (e.g., `linux-cachyos`).
+    ```shell
+    # /etc/default/limine
+
+    # Default parameters for most kernels
+    KERNEL_CMDLINE[default]="quiet splash rd.udev.log_priority=3"
+
+    # Specific parameters for the 'linux-cachyos' kernel
+    KERNEL_CMDLINE["linux-cachyos"]="quiet splash mitigations=off"
+
+    # Parameters for fallback entries (if generated)
+    # KERNEL_CMDLINE[fallback]="..."
+    ```
+2.  **Apply the changes:** After saving `/etc/default/limine`, regenerate your initramfs images and update the Limine entries. This incorporates your changes into the boot configuration file (`/boot/limine.conf`) automatically. Run:
+    ```bash
+    sudo limine-mkinitcpio
+    ```
+
+#### BIOS Systems
+
+On legacy BIOS systems, the `limine-entry-tool` utility has limitations and is not applicable for modifying kernel parameters through `/etc/default/limine`. Therefore, you need to **manually edit the Limine configuration file**.
```

Comment:
Looks like BIOS support was added recently, so we can use only limine-entry-tool for kernel parameters.

https://gitlab.com/Zesko/limine-entry-tool/-/commit/d0ffeb2c5475546038cb1a63246927d459d78feb